### PR TITLE
LIVE-1819 FTS Support

### PR DESCRIPTION
**Pull request by juan-cortes**
### Type

Feature

### Context

https://ledgerhq.atlassian.net/browse/LIVE-1819

No image or video since we're just using the nano s illustrations and this is for development purposes.

### CODE REVIEW

**Comment by juan-cortes** [comment]
Should be ready for testing again now that we solved the v3 dependency, @alalmi-ledger

**Review by alalmi-ledger — COMMENTED** [review]
QA OK

File: src/screens/PairDevices/Scanning.tsx [hunk 1]
```diff
@@ -32,11 +33,17 @@ export default function Scanning({ onTimeout, onError, onSelect }: Props) {
   const renderItem = useCallback(
     ({ item }) => {
       const knownDevice = knownDevices.find(d => d.id === item.id);
+      let modelId = "nanoX" as DeviceModelId;
+      if (item?.serviceUUIDs.length) {
```
Comment (lambertkevin) [inline src/screens/PairDevices/Scanning.tsx:37]: Shouldn't you add a `?` optionnal chaining on length ? If `serviceUUIDs` is undefined, should be problematic to get the length, isn't it ?

**Review by lambertkevin — APPROVED** [review]
Just 1 comment, but other than that LGTM 👌

**Pull request merged**